common.neon

```diff
@@ -5,17 +5,20 @@ services:
 			- addExtraModule(@Mathematicator\Engine\SampleModule)
 	mathematicator.queryNormalizer:
 		factory: Mathematicator\Engine\QueryNormalizer
-	mathematicator.translator:
-		factory: Mathematicator\Engine\Translator
```

Comment:
I think Mathematicator should contain basic Translator with dependencies to `Localization` (my abstract package) https://github.com/baraja-core/localization.

In math strings Translator must validate and rewrite LaTeX convention for each country specification (for example Czech republic use `3,14`, but USA use `3.14`).

Reply:
No problem, could you please implement it as you want?

Reply:
Reverted.